evergreen/config.yml

```diff
@@ -899,6 +967,31 @@ tasks:
       verbose_test_output: true
   - func: "check branch state"
 
+- name: baas-network-tests
+  # Uncomment once tests are passing
+  # tags: [ "for_nightly_tests" ]
```

Comment:
shouldn't this be uncommented?

Reply:
These tests are disabled for now until some of the failures can be addressed.

Reply:
is this the entire suite?

Reply:
It is the primary task that is used by both tests - so both tests are disabled.
The test parameters (non-ideal vs faulty networks) are configured by the variant for that test.